src/Provider/ImageProvider.php

```diff
@@ -88,7 +90,7 @@ public function getHelperProperties(MediaInterface $media, $format, $options = [
         $params = [
             'alt' => $media->getName(),
             'title' => $media->getName(),
-            'src' => $this->generatePublicUrl($media, $format),
+            $attrPrefix . 'src' => $this->generatePublicUrl($media, $format),
```

Comment:
This will render all images without an `src` tag, which is mandatory!
https://www.w3.org/MarkUp/html3/img.html